fe/fe-core/src/main/java/com/starrocks/system/HeartbeatMgr.java

```diff
@@ -318,9 +318,9 @@ public HeartbeatResponse call() {
                         Strings.isNullOrEmpty(e.getMessage()) ? "got exception" : e.getMessage());
             } finally {
                 if (ok) {
-                    ClientPool.heartbeatPool.returnObject(beAddr, client);
+                    ClientPool.beHeartbeatPool.returnObject(beAddr, client);
```

Comment:
for FE, seems it uses a new connection every time, not a connection pool, and also has a fixed connection and read timeout, do we need to change that?
```
                String result = Util.getResultForUrl(url, null, 2000, 2000);
```

Reply:
There is no need to use connection pool for HTTP connection, because it's short connection. I have changed the timeout to `Config.heartbeat_timeout_second`.